stream_interfaces.go

```diff
@@ -20,56 +20,99 @@ package grpc
 
 // ServerStreamingClient represents the client side of a server-streaming (one
 // request, many responses) RPC. It is generic over the type of the response
-// message. It is used in generated code.
+// message. If an error occurs on the stream, it will be returned as an instance
+// of the status package. Refer to the status package documentation for more
+// details. It is used in generated code.
```

Comment:
nit: `It is used in generated code` should remain wherever it is right now. `If an error occurs on the stream, it will be returned as an instance
// of the status package. Refer to the status package documentation for more
// details.` can be added in separate line

Reply:
Please do this everywhere